### ISSUE TITLE
Update the accessibility page in the theme developer handbook

### ISSUE THREAD

**Issue by carolinan** [WordPress/Documentation-Issue-Tracker#654]
## Issue Description

During a conversation in the WordPress Slack channel @joedolson requested an update to the Accessibility page in the theme developer handbook.

The menu item should also be moved so that it is more visible, and not hidden under classic themes -> functionality.

## URL of the Page with the Issue

https://developer.wordpress.org/themes/functionality/accessibility/

## Section of Page with the issue

Full page.

## Why is this a problem?

It is out of date, contains broken links and some opinionated content.

## Suggested Fix

An update to the same page has been written, but not yet published.
It needs to be reviewed, adjusted and then published.
https://developer.wordpress.org/wp-admin/post.php?post=11377&action=edit

**Comment by zzap** [comment]
Heads up @WordPress/docs-issues-coordinators, we have a new issue open. Time to use 'em labels.

**Comment by joedolson** [comment]
Happy to help review, although I don't have access to developer.wordpress.org to view the draft.

**Comment by zzap** [comment]
@joedolson you should have enough permissions now. Thank you.

**Comment by carolinan** [comment]
Thank you!

**Comment by joedolson** [comment]
I'm still getting a message that I don't have access to edit the in progress draft (https://developer.wordpress.org/wp-admin/post.php?post=144271&action=edit)

**Comment by zzap** [comment]
@joedolson It could be due to permissions for the Revisions plugin. I made you an administrator. Please, try again. Thank you

**Comment by carolinan** [comment]
Thank you, I have update the text.
I am not sure if I got the hyphens / dashes correct.

**Comment by joedolson** [comment]
I just made one small edit to collapse a space in `alt =""` and replaced one hyphen with an emdash; but otherwise I think this is good to go! Thanks!

**Comment by carolinan** [comment]
Thank you,
I have published the update, moved the menu item, and added the block theme accessibility page as a sub-menu item to that main page.
